test/workflow/classic_developer.sh

```diff
@@ -18,7 +18,7 @@ fairroot_branch=dev
 case "$ENVNAME" in
   *jun19*)
     fairroot_branch=v18.6_patches
-    no_boost_cmake=1
+    extra_cmake_flags="-DBoost_NO_BOOST_CMAKE=ON -DUSE_DIFFERENT_COMPILER=ON"
```

Comment:
Why is it needed?

Reply:
TBH, I have just refactored this. I haven't looked deeply enough, wether `USE_DIFFERENT_COMPILER` or the boost flag are actually needed for 18.6.

I just realized, that FairRoot/dev doesn't need the `USE_DIFFERENT_COMPILER` one.

Reply:
I remember adding support for homebrew and ccache compiler wrappers in v18.6.0 (e21d430bf5cb7a6e2f7b1fc0d70523d0f8f81cf5). Not sure about spack though.

Reply:
One possible way of thinking here might be:
The spack world is compiled with the spack wrapper.
But the workflow uses the normal compiler (and the user is responsible for choosing matching flags).
So compiler match checking isn't really easy.

Reply:
Can you please propose something to do next regarding this?

Reply:
I don't think there is anything to do.